common/djangoapps/util/tests/test_db.py

```diff
@@ -31,65 +27,6 @@ class TransactionManagersTestCase(TransactionTestCase):
 
     To test do: "./manage.py lms --settings=test_with_mysql test util.tests.test_db"
```

Comment:
@symbolist the tests below in this file related to the `outer_atomic` "decorator/context manager" functionality include calls to `unittest.SkipTest()`. In these tests, `outer_atomic` is called without the `read_committed` parameter (no isolation level change involved), but they are marked as applicable only for MySQL database engine, So I think we should keep the `unittest.SkipTest()` statements. Does it make sense for you?

Regarding the comment on how to run the tests, the problem is that the file `lms/envs/test_with_mysql.py` does not exist anymore (it was removed [here](https://github.com/edx/edx-platform/pull/17088) ) which make me think this test file just fell into oblivion. Do you think is it worth making an effort to include these tests in the edxapp CI?

Reply:
@symbolist it would be great since I'm touching code that is not being tested (tests are skipped according to [this](https://build.testeng.edx.org/job/edx-platform-python-pipeline-pr/32982/testReport/common.djangoapps.util.tests.test_db/TransactionManagersTestCase/) ). However, I don't know how much effort it would take since I'm not super skilled in testing. We are now more confident in this PR since you managed to run the tests with MySQL (btw, how did you figure it out??) and I confirmed the results. I don't think the inclusion of these tests in the CI suite is a critical blocker to get this PR merged. do you agree? cc @adzuci

Reply:
@jfavellar90 We do not need to do any further testing. The decorator is already being used in the code base and the remaining tests are passing with sqlite (if we remove the `unittest.SkipTest()` calls) so we just need to enable them to run as part of the standard test runs. We should do that as part of this PR. 🙂

Reply:
@symbolist sorry I thought outer_atomic couldn't be tested with SQLite. Changes already addressed, and tests running properly